kuma/javascript/src/last-modified-by.jsx

```diff
@@ -0,0 +1,36 @@
+// @flow
+import * as React from 'react';
+
+import { gettext } from './l10n.js';
+
+import ClockIcon from './icons/clock.svg';
+
+type LastModifiedByProps = {
+    lastModifiedBy: String,
+    lastModified: String,
+    profileBaseURL: String
+};
+
+export default function LastModifiedBy({
+    lastModifiedBy,
+    lastModified,
+    profileBaseURL
+}: LastModifiedByProps) {
+    return (
+        <section className="contributors-sub">
+            <ClockIcon />
+            <header>
+                <h4>{gettext('Last updated by:')}</h4>
+            </header>
+            <a href={`${profileBaseURL}${lastModifiedBy}`} rel="nofollow">
+                {`${lastModifiedBy}, `}
+            </a>
+            <time dateTime={lastModified}>
+                {new Date(lastModified)
+                    .toISOString()
+                    .slice(0, -5)
+                    .replace('T', ' ')}
```

Comment:
On the old site:
```
 Last updated by: estelle, Apr 25, 2019, 10:30:38 AM
```
On the new site:
```
Last updated by:estelle, 2019-04-25 14:30:38
```
(ignore the whitespace)

Hmm... First of all, I generally prefer ISO dates. But...

Now I don't know when estelle made this change because there's no timezone information. Suppose that there was timezone information I might not have the mental capacity to figure out when that was. E.g. "2019-07-11T19:30:25.648Z" I think means 19:00 minus 6 hours so 3pm here on US eastern. Ugh. Tired. 

We *could* use `Dateobj.toString()` but then you'd get something like this:
```
Thu Jul 11 2019 15:31:32 GMT-0400 (Eastern Daylight Time)
```
That looks visually appealing to my eyes but it's going to end up always being in the timezone of the SSR server (US Pacific) and in a US-centric internationalization format. 

My preferred solution to these things is usually to do something like GitHub does:
```
<time title="2019-07-09T19:30:25.648Z">2 days ago</time>
```
which is super easy to parse/read and understand and if you so desperately need to know exactly the full ISO timezone-aware string is there. 

I would be happy to NOT block this PR just because of this but instead copy my comments into a fresh new bug and from there drag in more people for feedback (or bikeshedding).

Reply:
PS. I looked into my MySQL database. estelle made that change at 10:30 pacific time which means 13:30 by time (because I'm US Pacific + 3h). 
So, right now I have no idea where this 14:30 comes from.

Reply:
Oh I get it now! 
What your code is *trying* to display to me is the GMT time. 
```
new Date("2019-04-25T10:30:38.382093")
Date Thu Apr 25 2019 10:30:38 GMT-0400 (Eastern Daylight Time)

new Date("2019-04-25T10:30:38.382093").toISOString()
"2019-04-25T14:30:38.382Z"
```
That's incorrect. It's not the truth. The best solution now would be to try to display it in a somewhat realistic fashion. Perhaps best to just show it like the current wiki does. Then we can worry about it after this PR lands.

Reply:
Urgh. This is hard that I thought. In the wiki, the date formatting is done with [`babel`](https://pypi.org/project/django-babel/). Something like this:

```python
from babel import dates
# tzvalue is the settings.TIME_ZONE + the MySQL naive datetime.
# locale is the locale of the page (obiously)
return dates.format_datetime(tzvalue, locale=locale)
```

So we need to try to do something similar. I.e. we need the supply the document locale to JavaScript's date formatting function. So far so good:

```
d = new Date()
Date Thu Jul 11 2019 16:30:39 GMT-0400 (Eastern Daylight Time)

d.toLocaleDateString('sv-SE')
"2019-07-11"
d.toLocaleString('sv-SE')
"2019-07-11 16:30:39"
d.toLocaleString('fr')
"11/07/2019 à 16:30:39"
d.toLocaleString('en-US')
"7/11/2019, 4:30:39 PM"
```
But first we need to convert our stupid naive datetime string into a timezone aware one.

Reply:
The thing is, the timezone information is lost in the JSON document API. I can find it out when I'm inside Django by looking at `django.conf.settings.TIME_ZONE`. 

"So, it's 'US/Pacific' you say. Ok, that's -7h from UTC. Everyone knows that."
If you're equipped with this "-7" fact you can do this: 
```javascript
var d = new Date(`${lastModified}-07:00`);
d.toISOString()
"2019-04-25T17:30:38.382Z"
```

And that's correct. I checked. 10:30 US Pacific is 17:30 in UTC.